packages/core/src/requestPool/imageRetrievalPoolManager.ts

```diff
@@ -8,7 +8,7 @@ import { RequestPoolManager } from './requestPoolManager';
  *
  * Retrieval (usually) === XHR requests
  */
-const imageRetrievalPoolManager = new RequestPoolManager();
+const imageRetrievalPoolManager = new RequestPoolManager('imageRetrievalPool');
```

Comment:
I must of missed this, why do we have an `imageLoadPool` and an `imageRetrievalPool` manager?

Reply:
It has changed for a long time, loading and retrieval has been separated to two different pools since we want to send request as soon as possible, but decoding is another story